website/assets/scss/common/_base.scss

```
@@ -7,10 +7,8 @@
  * 2. Basic image adjustments
  */
 
-@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap');
-
 html, body {
-  font-family: 'Inter var', 'Inter', system-ui, sans-serif;
+  font-family: 'Inter Variable', 'Inter var', 'Inter', system-ui, sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)